src/main/java/Duke.java

```diff
@@ -1,10 +1,85 @@
+import java.util.Scanner;
+import java.util.ArrayList;
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke \nWhat can I do for you?");
+        Scanner in = new Scanner(System.in);
+        ArrayList<Task> taskList = new ArrayList<Task>();
+
+        String userInput;
+        while(in.hasNext()){
+
+            userInput = in.nextLine();
+            if(userInput.equals("bye")){
+                break;
+            }
+            if(userInput.equals("list")) {
+                for (int i = 0; i < taskList.size(); i++) {
+                    System.out.print(i + 1 + ".");
+                    System.out.println(taskList.get(i).toString());
+                }
+            } else if (userInput.contains("todo")) {
+                String info = userInput.substring(5).trim();
+                taskList.add(new Todo(info));
+                int finalTask = taskList.size() - 1;
+                System.out.println("Got it. I've added this task:");
+                System.out.println(taskList.get(finalTask).toString());
+                if (taskList.size() == 1){
+                    System.out.println("Now you have " +taskList.size()+" task in the list.");
+                } else {
+                    System.out.println("Now you have " + taskList.size() + " tasks in the list.");
+                }
```

Comment:
You can consider refactoring this portion of code into a method as it is repeated multiple times in the if-else statements